File: Samples-Xamarin/MainActivity.cs
using System;

using Android.App;
using Android.Content;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.OS;
using Android.Support.V7.App;

namespace Samples
{
	[Activity (Label = "@string/title_activity_main", MainLauncher = true, Icon = "@drawable/icon")]
	public class MainActivity : AppCompatActivity
	{
		private ListView listControls;

		protected override void OnCreate (Bundle bundle)
		{
			base.OnCreate (bundle);
			SetContentView (Resource.Layout.Main);

			this.listControls = (ListView) this.FindViewById(Resource.Id.listControls);
			this.listControls.Adapter = new ControlsAdapter (this, 0);
			this.listControls.ItemClick += (object sender, AdapterView.ItemClickEventArgs e) =>  {
				ControlActivity.selectedControl = (ExamplesProvider)listControls.Adapter.GetItem(e.Position);

				Intent examplesActivity = new Intent(this, typeof(ControlActivity));
				this.StartActivity(examplesActivity);
			};
		}
	}
}

File: Samples-Java/Samples/src/main/java/activities/MainActivity.java
package activities;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ListView;

import com.telerik.android.sdk.R;


public class MainActivity extends AppCompatActivity {

    private ListView listControls;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        this.listControls = (ListView) this.findViewById(R.id.listControls);
        this.listControls.setAdapter(new ControlsAdapter(this, 0));
        this.listControls.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            @Override
            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                ControlActivity.selectedControl = (ExamplesProvider)listControls.getAdapter().getItem(position);

                Intent examplesActivity = new Intent(MainActivity.this, ControlActivity.class);
                MainActivity.this.startActivity(examplesActivity);
            }
        });

    }


}
